db/migrate/20161109150714_add_templates_to_resolve_url.rb

```diff
@@ -0,0 +1,21 @@
+class AddTemplatesToResolveUrl < ActiveRecord::Migration[5.0]
+  def up
+    Agents::WebsiteAgent.find_each do |agent|
+      keys = agent.event_keys
+      next if keys.nil? || !keys.include?('url') || agent.options.key?('template')
+      agent.options['template'] = keys.each_with_object({}) { |key, template|
+        case key
+        when 'url'
+          template[key] = '{{ url | to_uri: _response_.url }}'
+        else
+          template[key] = "{{ #{key} }}"
```

Comment:
On second thought, I think we should change how `template` works as you say, because the most typical use case would be transform just a couple of extracted values, like resolving `url`.  It is cumbersome and awkward to have to dead-copy keys from `extract` to `template` just to pass the other values through.